packages/office-ui-fabric-react/src/components/Nav/Nav.base.tsx

```diff
@@ -48,6 +49,7 @@ export class NavBase extends BaseComponent<INavProps, INavState> implements INav
     groups: null
   };
 
+  private _focusZone = createRef<IFocusZone>();
   constructor(props: INavProps) {
     super(props);
 
```

Comment:
If this is meant to be an exact port over, then initializeComponentRef(this) is missing right here

Reply:
This should not be an exact port, because it's to version 5 which still uses BaseComponent.